ePBFT/src/pbft/libbyz/Replica.cpp

```diff
@@ -253,20 +253,15 @@ bool Replica::compare_execution_results(
   return true;
 }
 
-size_t Replica::ledger_cursor() const
-{
-  return ledger_replay->cursor();
-}
-
 bool Replica::apply_ledger_data(const std::vector<uint8_t>& data)
 {
   PBFT_ASSERT(
     !data.empty(), "apply ledger data should not receive empty vector");
 
-  auto executable_pp =
+  auto executable_pps =
     ledger_replay->process_data(data, rqueue, brt, ledger_writer.get());
 
-  if (executable_pp)
+  for (auto& executable_pp : executable_pps)
   {
     auto seqno = executable_pp->seqno();
 
```

Comment:
A few lines down, if `!batch_info`, we should add `LOG_FAIL_FMT()` before returning.

Reply:
The method that is called to check batch info does LOG_FAIL if the info doesn't match and provides more detail as to which bit in ByzInfo didn't match.